apps/web/pages/team/[slug].tsx

```diff
@@ -35,6 +36,8 @@ function TeamPage({ team }: TeamPageProps) {
   const telemetry = useTelemetry();
   const router = useRouter();
 
+  const md = new MarkdownIt();
```

Comment:
This will create a new Markdown instance on each render. This is a no-no for performance.